modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/BulkRequestProcessor.java

```diff
@@ -1,201 +0,0 @@
-/*
-*  Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
-*
-*  WSO2 Inc. licenses this file to you under the Apache License,
-*  Version 2.0 (the "License"); you may not use this file except
-*  in compliance with the License.
-*  You may obtain a copy of the License at
-*
-*    http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing,
-* software distributed under the License is distributed on an
-* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-* KIND, either express or implied.  See the License for the
-* specific language governing permissions and limitations
-* under the License.
-*/
-package org.wso2.charon3.core.protocol;
-
-
-import org.wso2.charon3.core.exceptions.BadRequestException;
-import org.wso2.charon3.core.extensions.UserManager;
-import org.wso2.charon3.core.objects.bulk.BulkRequestContent;
-import org.wso2.charon3.core.objects.bulk.BulkRequestData;
-import org.wso2.charon3.core.objects.bulk.BulkResponseContent;
-import org.wso2.charon3.core.objects.bulk.BulkResponseData;
-import org.wso2.charon3.core.protocol.endpoints.GroupResourceManager;
-import org.wso2.charon3.core.protocol.endpoints.ResourceManager;
-import org.wso2.charon3.core.protocol.endpoints.UserResourceManager;
-import org.wso2.charon3.core.schema.SCIMConstants;
-
-/**
- *
- */
-public class BulkRequestProcessor {
```

Comment:
Removing public classes are not recommended.  There may be users who use this class. Changes should be done in a backward-compatible manner.